src/diffpy/structure/atom.py

```diff
@@ -224,9 +310,34 @@ def xyz_cartn(self, value):
     # anisotropy
 
     def _get_anisotropy(self):
+        """
+        Obtain the anisotropic thermal displacements parameters ``anisotropy``.
+
+        Parameters
+        ----------
+        None.
+
+        Returns
+        -------
+        self._anisotropy : Boolean
+            flag for anisotropic thermal displacements parameters. 
```

Comment:
Remove trailing space on line 323.  Fix all other space issues reported by
```
git diff --check origin/python3
```
Consider activating the `pre-commit` hook in your working repository to check for space issues prior to every commit:

```sh
cd .git/hooks
cp pre-commit.sample pre-commit
```